supplementary_style_guide/glossary_terms_conventions/general_conventions/e.adoc

```diff
@@ -37,6 +37,24 @@
 
 *See also*:
 
+[discrete]
```

Comment:
@dfitzmau can you remove the [discrete] tags in this PR? We no longer use this in our glossary entries (see #365)

Reply:
I accidentally deleted my branch located during a clean-up operation. Can I raise a separate PR to remove the discrete tags in the `e.adoc` file?

Reply:
Yep if there is no other feedback that is totally fine! I just wanted to make sure not to forget about it

Reply:
Hi @bergerhoffer . PR is merged. I created https://github.com/redhat-documentation/supplementary-style-guide/pull/367 . Would you be OK to merge or is the 3 approvals required?